src/client/game/scripting/function_tables.cpp

```diff
@@ -1872,7 +1872,7 @@ namespace scripting
 		{"_meth_843f", 0x843F}, // SP 0x1405D92F0 MP 0x14032D4E0
 		{"_meth_8440", 0x8440}, // SP 0x1404639C0 MP 0x140562210
 		{"_meth_8441", 0x8441}, // SP 0x140462EA0 MP 0x1405616E0
-		{"_meth_8442", 0x8442}, // SP 0x1402A36A0 MP 0x140358B60
+		{"linkto", 0x8442}, // SP 0x1402A36A0 MP 0x140358B60
```

Comment:
Don't we already have this? Correct me if wrong.

Reply:
No, was missing